packages/gatsby/src/bootstrap/load-config/index.ts

```diff
@@ -1,29 +1,23 @@
 import reporter from "gatsby-cli/lib/reporter"
```

Comment:
This file was renamed from `load-config-and-plugins` to `load-config/index.ts` and now exports `loadConfig`